### PR TITLE
Remove avalanche bootstrapping -> avalanche consensus transition

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

Now that Cortina has been activated, Avalanche bootstrapping should always result in accepting the stop vertex. This means that that Avalanche engine is completely inaccessible.

## How this works

- Changes the termination condition of Avalanche bootstrapping to when the stop vertex is accepted.
- Removes usage of the Avalanche consensus engine.

## How this was tested

CI + manually.

### CODE REVIEW

File: chains/manager.go [hunk 1]
```diff
@@ -921,47 +919,20 @@ func (m *manager) createAvalancheChain(
 	}
 
 	// create bootstrap gear
+	_, specifiedLinearizationTime := version.CortinaTimes[ctx.NetworkID]
```
Comment (StephenButtolph) [inline chains/manager.go:922]: We should probably force-ably linearize non- X-chain chains as well (regardless of the networkID)

File: snow/engine/avalanche/bootstrap/bootstrapper.go [hunk 2]
```diff
@@ -614,39 +609,18 @@ func (b *bootstrapper) checkFinish(ctx context.Context) error {
...
+		return err
 	}
 
-	// Notify the subnet that this chain is synced
-	b.Config.BootstrapTracker.Bootstrapped(b.Ctx.ChainID)
 	b.processedCache.Flush()
-
-	// If the subnet hasn't finished bootstrapping, this chain should remain
-	// syncing.
-	if !b.Config.BootstrapTracker.IsBootstrapped() {
```
Comment (gyuho) [inline snow/engine/avalanche/bootstrap/bootstrapper.go:639]: We don't need this anymore?

Comment (StephenButtolph) [reply snow/engine/avalanche/bootstrap/bootstrapper.go:639]: Nope! This is handled in snowman bootstrapping now.

File: snow/engine/avalanche/bootstrap/bootstrapper.go [hunk 3]
```diff
@@ -614,39 +609,18 @@ func (b *bootstrapper) checkFinish(ctx context.Context) error {
...
-		return b.Restart(ctx, true)
+	// Invariant: edge will only be the stop vertex after its acceptance.
+	edge := b.Manager.Edge(ctx)
+	stopVertexID := edge[0]
+	if err := b.VM.Linearize(ctx, stopVertexID); err != nil {
+		return err
 	}
 
-	// Notify the subnet that this chain is synced
-	b.Config.BootstrapTracker.Bootstrapped(b.Ctx.ChainID)
```
Comment (danlaine) [inline snow/engine/avalanche/bootstrap/bootstrapper.go:634]: This isn't needed because it's done in snowman's bootstrapper now, right?

**Review by danlaine — COMMENTED** [review]
just 2 questions

File: snow/engine/avalanche/bootstrap/bootstrapper.go [hunk 4]
```diff
@@ -319,16 +286,8 @@ func (b *bootstrapper) Disconnected(ctx context.Context, nodeID ids.NodeID) erro
...
-	if !b.awaitingTimeout {
-		return errUnexpectedTimeout
-	}
-	b.awaitingTimeout = false
-
-	if !b.Config.BootstrapTracker.IsBootstrapped() {
-		return b.Restart(ctx, true)
-	}
-	return b.OnFinished(ctx, b.Config.SharedCfg.RequestID)
+func (*bootstrapper) Timeout(context.Context) error {
```
[the remark this answers is not in the document]
Comment (StephenButtolph) [reply snow/engine/avalanche/bootstrap/bootstrapper.go:289]: This is still used in the snowman bootstrapper - so we can't remove it from the interface.

**Review by danlaine — APPROVED** [review]

File: chains/manager.go [hunk 5]
```diff
@@ -976,7 +948,7 @@ func (m *manager) createAvalancheChain(
 		Avalanche: &handler.Engine{
 			StateSyncer:  nil,
 			Bootstrapper: avalancheBootstrapper,
-			Consensus:    avalancheEngine,
+			Consensus:    nil,
```
Comment (StephenButtolph) [inline chains/manager.go:951]: This is currently broken. This doesn't support bootstrapping the DAG after bootstrapping has finished.

**Review by danlaine — APPROVED** [review]

**Review by gyuho — APPROVED** [review]

**Pull request merged**